packages/server-wallet/src/wallet/__test__/integration/create-channel.test.ts

```diff
@@ -38,7 +37,7 @@ describe('happy path', () => {
               signedStates: [{turnNum: 0, appData}],
               objectives: [
                 {
-                  participants: [alice(), bob()],
+                  participants: [], // todo: remove, currently deprecating participants
```

Comment:
Can we use all caps `TODO:`? I use [this extension](https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight) to highlight these things